.github/workflows/ci.yml

```diff
@@ -140,24 +140,12 @@ jobs:
         run: ceedling project:macos_arm64 verbosity[4] clobber dependencies:make
       - name: Run build
         run: ceedling project:macos_arm64 verbosity[4] release
-  windows-64bit:
-    runs-on: windows-2022
-    steps:
-      - uses: actions/checkout@v3
-      - name: Install Ceedling
-        run: gem install ceedling --no-user-install
-      - name: Check Ceedling version
-        run: ceedling version
-      - name: Set up Visual Studio shell
-        uses: egor-tensin/vs-shell@v2
-        with:
-          arch: x64
-      - name: Build dependencies
-        run: ceedling project:windows_64 verbosity[4] clobber dependencies:make
-      - name: Run build and test
-        run: ceedling project:windows_64 verbosity[4] test:all
-  windows-32bit:
+  windows:
     runs-on: windows-2022
+    strategy:
+      fail-fast: true
```

Comment:
Is it intentional to fail-fast ? It will cancel all in-progress and queued jobs if one failed

Reply:
I meant to write false, good catch